### DISCUSSION TITLE
無痕模式下的網站瀏覽和活動其實仍被Google記錄

### DISCUSSION THREAD

**Discussion by alvin119** [1101-GS4719-C/Python-Community-GS4719-C#1833] [General]
原本我也以為在使用無痕模式時活動是不會被追蹤的，
所以以前不想要網頁出現瀏覽過的商品或是相關的廣告的時候，就會開無痕模式。
不過現在看來Google仍然會追蹤用戶的活動來計算網站流量跟販賣廣告，
感覺Google侵犯了我的數據隱私。
大家對這件事是怎麼看的?
資料來源 : https://infosecu.technews.tw/2024/01/03/google-user-privacy/

**Comment by Patsy17** [comment]
感覺現在做任何事情都會被監視，
就連有時候跟朋友聊天時提到想要買什麼商品，
沒多久我的推薦廣告都會跑出相關的商品！
真的好恐怖～～～

**Comment by FakeWood** [comment]
這邊推薦大家一個免費的瀏覽器，叫做duckduckgo，有興趣的可以搜尋一下，他主打的就是不會收集用戶資料，還會幫你擋網頁追蹤器，保護你的隱私，不過我還是很好奇這樣duckduckgo到底要靠甚麼賺錢就是了

**Comment by TobywithDino** [comment]
竟然會偷看我的瀏覽紀錄QQ Google超壞，這樣我想瑟瑟的時候都很怕被google追蹤ㄟ 希望他們可以正視一下使用者的隱私問題

**Comment by chuanmin0317** [comment]
以前就有聽過類似的事情了，不過因為有時候還是不想讓一些搜尋紀錄出現影響到演算法，所以還是會開無痕模式，不過我覺得google還是應該尊重用戶不想讓其他人知道它的搜尋紀錄的權力，不應該隨意收集無痕模式的紀錄

**Comment by BeaGle0928** [comment]
我有看過一張梗圖是Google在你沒打開麥克風的時候，當你對著麥克風講話，Google會跳出提示說「您可能忘記打開麥克風了喔」，仔細思考一下這樣我到底是有沒有關掉麥克風呢？

**Reply by yubo0oo** [reply]
我也有同樣的納悶
阿怎麼知道我在講話

**Comment by chiuwangwang** [comment]
我認為只要在網路上活動，就一定會留下痕跡，所以我認為未來，大家直接不裝了，問多少賣自己的隱私權，但時候我可定會想辦法拉高自己的價值。

**Comment by lyykey** [comment]
很多時候，我們無痕就是為了做壞壞的事情，不想被其他人發現你的瀏覽紀錄，但事實上，只要我們用電腦就會被正在使用的伺服器站存瀏覽資料，所以即使開無痕我也知道自己正在被監視，但是其實只要抱著良好的心態，其實這些監視都還好，因為全世界瀏覽量那麼高，沒可能他們要特別關注自己吧，因為我不是柯震東

**Comment by benbebobo149** [comment]
你在開啟無痕模式的時候，他就有提示說無痕模式不是真的完全找不到任何紀錄，所以不用意外說你用無痕模式不會被記錄。所以我推薦使用 SurfShark VPN。我對這一提醒感到印象深刻，它強調了在線上保護隱私的重要性。使用 VPN 可以幫助我們加密網絡流量，隱藏 IP 地址，從而更有效地保護個人信息免受網絡監視和攻擊。我會把這個信息分享給我的朋友和家人，讓他們也意識到保護隱私的重要性。最後，我要感謝同學的分享，讓我有機會了解到 SurfShark VPN 這樣一個有用的工具，這將對我們的網絡安全有所幫助。

**Comment by Eileen050824** [comment]
雖然本來就知道在網路上瀏覽很容易留下紀錄，google的無痕分頁也有稍作提示，但現在大家都相當注意隱私的重要性，也希望系統方更加尊重使用者權益，當然使用者也不要做會令自己後悔的事，也算是自我保護的方式！

**Comment by YANRONG0123** [comment]
我認為公司為了存活，很多事情無法達到百分百的保證，而且我覺得無痕視窗，只是用來避免被生活周遭的人，探查到自己所搜尋的內容，只要不為非作歹，其實被公司知道沒什麼關係。

**Comment by YuanWang08** [comment]
我覺得無痕模式還追蹤用戶的瀏覽紀錄是一個很缺德的行為。我用無痕模式就是為了不希望我的瀏覽紀錄被拿去讓演算法運作。演算法真的很可怕，我之前在研究買新鍵盤的時候，頻繁的搜尋鍵盤，不用一天的時間facebook的廣告，蝦皮的廣告全都是鍵盤。但無痕模式似乎還是避免發生這種事，雖然我還是覺得他們有在偷存資料。

**Comment by yyujimmy** [comment]
我認為這件事情凸顯了數據隱私保護方面的一些挑戰和問題。雖然無痕模式通常被視為一種保護隱私的方法，但事實證明，即使在這種模式下，仍然存在被追蹤的風險。這使得用戶難以信任技術公司對他們個人數據的處理方式。

對我來說，這件事情強調了兩個關鍵問題。首先是透明度和信任。技術公司應該更加透明地向用戶解釋他們的數據收集和使用方式，讓用戶能夠清楚地知道自己的數據被用於何種目的。其次是數據隱私保護的法律和規定。政府應該制定更加嚴格的法律，以確保技術公司遵守數據隱私的最高標準

**Comment by KH1126** [comment]
我沒怎麼用過無痕，但我覺得開無痕主要是不想在本地設備上存取瀏覽網站時的紀錄。
所以在無痕模式下的網站瀏覽和活動仍被Google記錄好像影響不大?

**Comment by tzurou** [comment]
之前就有聽說過用無痕模式google還是會有我的瀏覽紀錄，我用無痕只是不想在我看的到的地方留下搜尋紀錄，這應該也是google無痕模式的意義，我自己覺得這樣沒關係對我來說除了google和我以外沒有第三人知道我用無痕搜尋什麼就好了。

**Comment by titan92601** [comment]
我認為確實是一個引起關注的議題。無痕模式通常被認為是一種保護隱私的工具，但如果其功能未能如預期般完全隱匿用戶活動，這就可能導致使用者的隱私受到侵犯。這也凸顯了數據隱私保護的重要性，以及對於科技公司如何處理用戶數據的關注。這樣的情況可能會引發用戶對於個人數據管理和隱私保護的更多關注和反思。

**Comment by alexender0104** [comment]
我覺得這是一件蠻可怕的事，無痕模式下的瀏覽仍然可能被Google記錄。無痕模式通常會阻止瀏覽器在本地存儲歷史記錄、Cookie和其他網站數據，但它並不會完全隱藏我的活動。我的網絡服務提供商（ISP）和網站本身仍然可以記錄的訪問活動。

**Comment by OHoln** [comment]
對我來說無痕模式的功能就只是隱藏瀏覽紀錄而已，所以好像也影響不大，而且我認為這些大公司真的想要你的數據也不缺這個手段，防不勝防。

**Comment by wanka2030** [comment]
無痕模式提供有限的隱私保護，不儲存本地瀏覽歷史和cookie，但無法完全匿名。Google仍可記錄登入帳戶的活動，網站追踪器、ISP及網絡管理員仍能監控你的行為。無痕模式無法隱藏IP地址。若需更高的匿名性，應考慮使用VPN或Tor瀏覽器，以更好地保護網上隱私。

**Comment by hbh10601** [comment]
無痕模式對我來說比較像是不想要影響演算法的一個模式，不過我也可以理解無法完全匿名這件事，如果有違法事實要追查都還是可以查的到

**Comment by sandyho0307** [comment]
我覺得Google應該透明地告知用戶他們的數據會如何被使用，並提供更多選擇來保護用戶的隱私。深為用戶的我們則應有權決定自己的數據是否被追蹤和使用，而不是被動地接受追蹤行為。

**Comment by 710388** [comment]
這是肯定的吧 ，無論何時網路資料都幾乎會被google 拿走，畢竟用的是他們瀏覽器

**Comment by cassy0201** [comment]
我前陣子在切換無痕模式的時候就有發現google有提醒如果要查紀錄還是查得到，只是在裝置上查不到，感覺以現在的技術只要在網路上活動就一定會留下痕跡

**Comment by yubo0oo** [comment]
現今的網路凡走過必留留下痕跡
請各位網路成癮症患者小心行事

**Comment by MingWei174** [comment]
當用戶使用Chrome瀏覽器的無痕模式時，以為自己的隱私得到保護，但其實谷歌仍會追蹤的瀏覽習慣。 
2020年有人指控谷歌對用戶進行不當追蹤，誤導使用者、侵犯其隱私，追蹤無痕瀏覽器用戶的搜尋紀錄、瀏覽活動等。 作為達成和解的一部份，谷歌同意刪除數十億筆他們收集的用戶私人瀏覽資料。
但我覺得他們要收集大數據資料來投放相對應的廣告來獲取利潤，追蹤每個使用者的瀏覽習慣是不可缺少的

**Comment by hunteryang0108** [comment]
感謝同學分享的資訊，不過我認為這個並不是Google Chrome在收集資訊，而是用戶使用的網站收集資訊後與Google共享，是網站經營者與Google互利共生的一種生態。只能說是Google這個事業體太過龐大，他們從各種方面蒐集使用者的資訊，以至於使用者會覺得自己的隱私權受到侵害。

**Comment by Nana1218** [comment]
我認為，凡走過必留下痕跡，即使是開了無痕模式，刪除了當下所有查詢的紀錄，但在現在網路的時代，任何痕跡都是可以被找出的，無論是利用修復或是尋找源頭，不可能做到完全無痕。

**Comment by kingsophy** [comment]
我認為無痕模式只是相對一般模式比較沒有痕跡，凡走過必留下痕跡，但現今社會想要不在網路上留下痕跡是不可能的事，只能想辦法減少去一些網站，只能說隱私在這個社會已經越來越廉價了。

**Comment by ds0w0** [comment]
確實，很多人曾經以為無痕模式能完全保護隱私，但事實上，它只是防君子不防小人。Google的行為確實引發了對隱私的關注，因為即使在無痕模式下，某些數據仍被記錄和使用。這顯示出了數位隱私問題的挑戰。我相信有些人可能會感到失望，因為他們相信無痕模式能夠保護他們的隱私，結果卻發現自己仍然被追蹤。但我覺得大部分人會認同，了解了這些信息後，可以更謹慎地使用網路服務和管理自己的數據。

這一事件確實提醒我們，在數位時代，保護隱私需要我們更加主動和警惕。除了了解不同模式和工具，像是Cookie，的功能和限制外，也應該關注公司如何處理我們的數據，並在使用數位服務時保持警覺。對於想要進一步保護上網隱私的用戶，使用 VPN 是一個不錯的選擇。VPN能加密你的網路流量，隱藏你的 IP 地址，並讓你在公眾網路上更安全地瀏覽網頁。這樣，即使在無痕模式下，線上活動也會受到額外的保護，避免被網站或第三方追蹤。
